model_service/models/llama2/model.py

```diff
@@ -215,49 +249,22 @@ def generate(self, request):
         results = response_object.json()
 
         # Compile the results into a structure consistent with other kaleidoscope models
-        activations = []
-        generated_sequences = []
+        activations = results["choices"][0]["activations"]
+        generated_sequences = GENERATOR.tokenizer.decode(results["choices"][0]["text"][0])
```

Comment:
```suggestion
        generated_sequences = GENERATOR.tokenizer.decode(results["choices"][0]["text"])
```
Why are we only extracting the first generated sequence? `len(tokens) == len(generated_sequences)` should hold right? Am I missing any logic here?

Reply:
That's just because all the sequences are wrapped in a list object under the hood. I don't know why Meta did it that way. If there are multiple generated sequences, that `[0]` element contains a list of all sequences.